### PR TITLE
Update profile daniel chen

### PR DESCRIPTION
**Pull request by chendaniely**

### CODE REVIEW

File: _includes/people/chen.daniel.html [hunk 1]
```diff
@@ -1,8 +1,14 @@
 <div class="media">
   <img class="media-object pull-left" src="{{page.root}}/img/people/chen.daniel.jpg" alt="Daniel Chen" />
   <div id="chen.daniel" class="media-body">
-    <span class="person">Daniel Chen</span>
-    completed his Masters in Public Health in the Department of Epidemiology at Columbia University Mailman School of Public Health.
-    He is hoping to combine scientific computing and data science with epidemiology to answer questions about population health.
+    <span class="person"><a href="http://chendaniely.github.io>Daniel</a> <a href='http://twitter.com/chendaniely'>Chen</a></span>
```
Comment (gvwilson) [inline _includes/people/chen.daniel.html]: You can use `<a class="person" href="...">...</a>` for this.

Comment (gvwilson) [inline _includes/people/chen.daniel.html]: Please list your twitter ID separately in parens after your name `(<a href="...">@...</a>)`

File: _includes/people/chen.daniel.html [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+    is a doctoral student in <a href='http://gbcb.vbi.vt.edu'>Genetics, Bioinformatics, and Computational Biology</a>
+    at Virginia Tech and currently working in the
+    <a href='http://vbi.vt.edu/sdal'>Social and Decision Analytics Laboratory</a>
+    under the <a href='http://vbi.vt.edu'>Virginia Bioinformatics Institute</a>.
+    He received his MPH at the
+    <a href='http://www.mailman.columbia.edu'>Mailman School of Public Health</a> in Epidemiology and
+    is interested in integrating hospital data in order to perform predictive health analytics
+    and build clinical support tools for clinicians.
   </div>
 </div>
```
Comment (gvwilson) [inline _includes/people/chen.daniel.html]: The new picture is *sharp* :-)

Comment (chendaniely) [inline _includes/people/chen.daniel.html]: bow ties are cool :D

**Pull request merged**